server/repository/src/migrations/v2_11_00/add_shipping_method_table.rs

```diff
@@ -0,0 +1,24 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "add_shipping_method_table"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+                CREATE TABLE shipping_method (
```

Comment:
And please move migrations to 2.12~ tyty

Reply:
Thanks Roxy, please see changes here: https://github.com/msupply-foundation/open-msupply/pull/9277/commits/edba4b1480d030ec3857ff7199b7010f15541b95